src/js/stores/AppFormStore.js

```diff
@@ -388,13 +388,15 @@ function processResponseErrors(responseErrors, response, statusCode) {
   } else if (statusCode === 422 && response != null &&
       Util.isArray(response.details)) {
     response.details.forEach(detail => {
-      var attributePath = detail.attribute.length
-        ? detail.attribute
-        : "general";
-      var fieldId = resolveResponseAttributePathToFieldId(attributePath) ||
-        attributePath;
-      responseErrors[fieldId] =
-        AppFormErrorMessages.lookupServerResponseMessage(detail.error);
+      var path = detail.path;
+      var fieldId = resolveResponseAttributePathToFieldId(path) || path;
+      if (detail.errors.length >= 1) {
+        responseErrors[fieldId] =
+          AppFormErrorMessages.lookupServerResponseMessage(detail.errors[0]);
```

Comment:
I thought about this, but couldn't come up with a way of making it look good. We don't have a guarantee of how long the error message will be or whether it will follow any specific format, so concatenating with blank strings could lead to running words together accidentally. We could handle rendering them into a list, but then we need to make sure we're happy with how this looks.

An example of a situation in which multiple errors are returned would really help in making this call.